extensions/amp-ad/0.1/amp-ad-custom.js

```diff
@@ -84,7 +84,9 @@ export class AmpAdCustom extends AMP.BaseElement {
     // If this promise has no URL yet, create one for it.
     if (!(fullUrl in ampCustomadXhrPromises)) {
       // Here is a promise that will return the data for this URL
-      ampCustomadXhrPromises[fullUrl] = xhrFor(this.win).fetchJson(fullUrl);
+      ampCustomadXhrPromises[fullUrl] = xhrFor(this.win).fetchJson(fullUrl, {
```

Comment:
@xgretsch check [this](https://github.com/ampproject/amphtml/blob/master/spec/amp-cors-requests.md#cors-security-in-amp) if you haven't. Because AMP can be cached, all pages from different publisher are served on the same CDN domain. So AMP CORS is an important mechanism to differentiate different publishers on CDN.